### PR TITLE
remove rare settings, move open settings to top

### PR DESCRIPTION
**Pull request by tybug**
Removed some settings that I think were just cluttering up the Settings tab. They're still editable from the settings.cfg file of course.

Also gave some emphasis to being able to open the settings file by placing it at the top. Doesn't look great without a separator at the top, but i think that's just because I'm used to there being one there.

### CODE REVIEW

File: circleguard/widgets.py [hunk 1]
```diff
@@ -318,19 +314,13 @@ def __init__(self, tooltip):
         output_combobox.setInsertPolicy(QComboBox.NoInsert)
         output_combobox.setCurrentIndex(0) # NONE by default
         self.output_combobox = output_combobox
-        self.save_folder = FolderChooser("Log Folder", get_setting("log_dir"))
```
Comment (InvisibleSymbol) [inline circleguard/widgets.py]: I agree with removing the `FolderChooser` but I would still like to have an easy way to toggle saving logs in the gui.

Comment (tybug) [reply circleguard/widgets.py]: for the general user or for development? I dot think we should really let people toggle logs or we might get someone reporting an issue, we ask for logs, and they don’t have any. It doesn’t do any harm to them if we save logs. 

For development I’m ok with adding it to the gui, just under the dev section.

Comment (InvisibleSymbol) [reply circleguard/widgets.py]: Well if the default is on then we don't really need the toggle, true.

**Comment by InvisibleSymbol** [comment]
> Also gave some emphasis to being able to open the settings file by placing it at the top. 

I find it weird that you want to have it at the top. Shouldn't it be at the bottom in a section called "Advanced" or something? Having the "open settings" above everything, even the api key, just feels wrong.

**Comment by tybug** [comment]
just wanted people to see it before anything else. If the first things they see are toggleable settings, they may not look any further and realize there's a ton of hidden and editable settings a click of a button away.

**Comment by InvisibleSymbol** [comment]
Well they are, as you wrote, _hidden_. Important and often used settings should be accessible from the gui, less used ones should be in the file. Which can be at the bottom of the tab, since you won't use those that often.

**Comment by tybug** [comment]
I don't want them to be hidden though, I want to make sure the user knows they exist. 

What if the open and sync settings buttons were at the top right (to the right of the version text)? I like that actually because they're not really a setting per se like the other ones are, so they're separated, but also clearly visible to a user just clicking around on the setting tab.

**Pull request merged**